### ISSUE TITLE
Select examples should have aria-selected="false" for non-selected items

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly#5799]
**Describe the enhancement or change**
Followup to https://github.com/patternfly/patternfly/pull/5663

The [Option single select example](https://www.patternfly.org/components/menus/menu/html#option-single-select) and [option multi select example](https://www.patternfly.org/components/menus/menu/html#option-multi-select) on the Menu HTML page need the following updates:

- `role="menu"` needs to be updated to `role="listbox"`

- `role="menuitem"` needs to be updated to `role="option"`

- Each role=option item needs to have an `aria-selected` attribute, either true or false. this cannot be omitted

- For the multi-select example, the role=listbox should have `aria-multiselectable="true"` as well.

We can update the a11y docs to include the aria attributes here as well, rather than adding them to the HTML "Accessibility" table (which would be removed anyway inf avor of the a11y docs). I'll open an issue in org for this.

**Is this request originating from a Red Hat product team? If so, which ones and is there any sort of deadline for this enhancement?**

**Any other information?**

---

**Jira Issue:** [PF-501](https://redhat.atlassian.net/browse/PF-501)

**Comment by mcoker** [comment]
ping @thatblindgeye to review and update the description.

**Comment by thatblindgeye** [comment]
Updated criteria in original comment
